packages/fetch/src/fetch.js

```diff
@@ -12,6 +12,7 @@ import zlib from 'zlib';
 import fs from 'fs';
 import * as mime from 'mrmime'
 import dataUriToBuffer from 'data-uri-to-buffer';
+import {Buffer} from 'buffer';
```

Comment:
Why introduce this dependency ? I don't see buffer used anywhere in this module.

Reply:
It is used in `fixResponseChunkedTransferBadEnding`

https://github.com/web-std/io/blob/c770c7ff33eb951c67677181b39c1afe525a9da0/packages/fetch/src/fetch.js#L327